components/provider_fs/src/lib.rs

```diff
@@ -74,7 +74,7 @@
 //!
 //! ```text
 //! cargo run
-//!   --features export-bin,bincode,serialize_none
```

Comment:
Nitpick: Please re-generate the test data in */resources/testdata*.

Reply:
Hmm, https://github.com/unicode-org/cldr-json/releases/download/37.0.0/cldr-37.0.0-json-full.zip no longer works; is there a more stable URL we should be using?

Reply:
Oh, I can pass in a newer version.

Reply:
```
thread 'main' panicked at 'not implemented: This skeleton string is not yet supported: "Md-alt-variant"', components/provider_cldr/src/transform/dates.rs:164:25
```

hmmmm

Reply:
Done!